html/changelogs/paradoxspace-t1000.yml

```diff
@@ -0,0 +1,7 @@
+author: ParadoxSpace & mikomyazaki
```

Comment:
is this even snakebitten's work at this point

Reply:
Yes, slightly. He did contribute and this PR would not have been made without him.